tests/Feature/Sensors/QuerySensorTest.php

```diff
@@ -17,6 +17,8 @@
 use MongoDB\Laravel\Connection as MongoDbConnection;
 use PDO;
 use PHPUnit\Framework\Attributes\DataProvider;
+use SingleStore\Laravel\Connect\Connection as LegacySingleStoreConnection;
+use SingleStore\Laravel\Connect\SingleStoreConnection;
```

Comment:
The class name was changed in `v2.0.0` and we need to test against both v1 and v2 because they support different Laravel versions.